opentelemetry/chart/values.yaml

```diff
@@ -44,10 +45,10 @@ openTelemetry:
   cluster:
   region:
   logsCollector:
-    enabled: false
+    enabled: true
   prometheus:
   podMonitor:
-    enabled: false
```

Comment:
The `logsCollector` and `podMonitor` are also options in the `PluginDefinition` where they are defaulted to true, false respectively.  These values in the `values.yaml` have no effect in this case and the values should be configured on the Plugin instead.

Reply:
I have removed these defaults.